packages/wallet/src/wallet.ts

```diff
@@ -1,28 +1,25 @@
 import { AdvanceRequestHandler, Voucher } from "@deroll/app";
-import {
-    Address,
-    encodeFunctionData,
-    erc20Abi,
-    getAddress,
-    isAddress,
-} from "viem";
-
-import { cartesiDAppAbi, dAppAddressRelayAddress } from "./rollups";
-import {
-    isERC20Deposit,
-    isEtherDeposit,
-    parseERC20Deposit,
-    parseEtherDeposit,
-} from ".";
+import { Address, getAddress, isAddress } from "viem";
+
+import { inspect } from "node:util";
+import { TokenHandler } from "./token";
 
 export type Wallet = {
     ether: bigint;
-    erc20: Record<Address, bigint>;
+    erc20: Map<Address, bigint>;
```

Comment:
Record is more appropriate than Map. Any reason of changing to Map?

Reply:
Changing to maintain consistency in other tokens, Like ERC721 and ERC1155 that use Map and Set.